src/Quasar/Mount/SparkHDFS.purs

```diff
@@ -54,7 +54,7 @@ fromJSON ∷ Json → Either String Config
 fromJSON
   = fromString
   <=< (_ .? "connectionUri")
-  <=< (_ .? "spark")
```

Comment:
This `spark` is not the URIScheme, which is listed below. I think this _should_ be `"spark-hdfs"` as that's the connector type, right?